testing/test_aux.out

```diff
@@ -36,5 +36,8 @@ sec.ofctl
   }
 }
 Redacted docs diff
-No report diff
+15c15
```

Comment:
This needs to be cleared up -- the report diff should be empty.

Reply:
Yeah, I need to get better at this!